aio/src/app/shared/select/select.component.spec.ts

```diff
@@ -1,189 +1,63 @@
-import { Component, DebugElement } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { By } from '@angular/platform-browser';
-import { SelectComponent, Option } from './select.component';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatSelectModule } from '@angular/material/select';
+import { SelectComponent } from './select.component';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import { HarnessLoader } from '@angular/cdk/testing';
+
+import { MatSelectHarness } from '@angular/material/select/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { MatFormFieldHarness } from "@angular/material/form-field/testing";
```

Comment:
```suggestion
import { MatFormFieldHarness } from '@angular/material/form-field/testing';
```
Looks like aio tests are failing lint checks because of this.

Reply:
cool yeah, sorry for the failing tests, I'm on it :slightly_smiling_face: